sb_vision/distance_finding.py

```diff
@@ -0,0 +1,121 @@
+"""
+Handlers for detecting 3D marker Location.
+
+Also handles loading the calibration file which is passed into the
+location finding function.
+"""
+
+import functools
+import re
+from pathlib import Path
+from typing import Any, Dict, List, NamedTuple, Tuple
+
+import cv2
+import numpy as np
+from lxml import etree
+
+Coordinate = NamedTuple('PixelCoordinate', [('x', float), ('y', float)])
+
+
+def _get_values_from_xml_element(element: etree.Element) -> List[str]:
+    """Parse an xml tag with space-separated variables."""
+    text = []  # type: List[str]
+    for e in element.itertext():
+        if e.strip():
+            text += re.split('\s+', re.sub('\n', '', e.strip()))
+    return text
+
+
+def get_calibration(file_name: Path) -> Dict[str, Any]:
+    """
+    Parse a calibration xml generated by the camera calibration tool.
+
+    (see https://docs.opencv.org/3.4.0/d7/d21/tutorial_interactive_calibration.html)
```

Comment:
It's great to have this link here, though we'll also need some wider documentation of this. Please could you update the calibration section of the README to match the new procedure? (A brief description then pointing at this link is fine; excellent would be also including brief instructions on validating the resulting calibration and mentioning which updates need to be made to include the calibration in the python/debian packages)